### DISCUSSION TITLE
Issue with stacking

### DISCUSSION THREAD

**Discussion by robby2629** [photoprism/photoprism#4605] [Q&A]
I just installed PhotoPrism and I understand from the FAQs that stacking cannot be completely disabled. I am sure the problem I am describing below is one many of you have come across and would appreciate some guidance.

Over the years I have accumulated photos from several iphones, DSLRs etc.  Specifically with the phones, each new phone started its file naming with IMG_0001. I now have about 50,000 pictures but there are now 100s or maybe 1000s where the file names are the same. For example, I have 4 instances of IMG_4400 but in different folders.

When I first indexed and initialized the DB, PhotoPrism improperly stacked these images even though they have nothing to do with each other. To circumvent this I used a script to rename all the files with IMG_XXXX-MM-YYYY (month and year suffix) so that Photoprism will not see them as files to be stacked. This didn't work because the exif data contains a field called "original filename" (which is unchanged) and Photoprism uses this for stacking.

Any suggestions on how I get around this?

**Comment by srett** [comment]
> When I first indexed and initialized the DB, PhotoPrism improperly stacked these images even though they have nothing to do with each other.

Is this ages ago, or on a recent/fresh install?
I have colliding filenames too, my DSLR rolled over a year ago, and it's using the default DCIM_XXXX.JPG pattern that a couple other cameras I used over the years/decades have, but PhotoPrism never stacked any of these together, as they sit in different folders. AFAIU, for stacking the files must sit in the same folder and start with the same basename, and also have matching timestamps, which they most likely don't in your case, I assume.

If you can reliably reproduce the issue, it might help if you turn the log level up to debug and post the log output of the index run where it does the stacking.

**Comment by lughino** [comment]
I do have the same problem. Over the years, migrating iphones, my iphone gallery contains lots of files named equal. Most of them are files in the same folder named the same but with different extension.
An example is a photo of two days ago and a `.mov` video of 4 years ago. these get stacked together. I also tried to rename the files like @robby2629 did, but I got the same result with a fresh new instance. It seems that the metadata is used and not the filename.

Any help would be very much appreciated.
I can provide some logs, but I would need to get pointed to a doc to do so as I turned the debug logs in the advanced menu, but the only logs I see are a lot of `webdav: IMG_0377.yml already exists`
